src/react-app-env.d.ts

```diff
@@ -1 +1,25 @@
 /// <reference types="react-scripts" />
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user: User | null,
+  comments: Comment[] | [],
```

Comment:
same here, empty array here is redundant

Reply:
Understood, thank you!